crt_portal/cts_forms/migrations/0049_fix_other_duplicates.py

```diff
@@ -6,7 +6,7 @@
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('cts_forms', '0046_update_primary_complaint'),
+        ('cts_forms', '0050_auto_20200226_1532'),
```

Comment:
I am getting an error running this migration.
```
raise InconsistentMigrationHistory(
django.db.migrations.exceptions.InconsistentMigrationHistory: Migration cts_forms.0049_fix_other_duplicates is applied before its dependency cts_forms.0050_auto_20200226_1532 on database 'default'.
```
 I don't think my DB and the dev DB that have already applied these [migrations up to migration 50](https://github.com/usdoj-crt/crt-portal/tree/develop/crt_portal/cts_forms/migrations), so I don't think we can change the order on the migrations that were already run. I think if you keep the order as it is in the develop branch, have your new migration fork from where you were, a new merge migration could get them synced up.  

Happy to connect if you want to pair on this tomorrow.

Reply:
Ah thanks, will fix this morning!

Reply:
Ooo now I'm remembering, the problem I was trying to solve here is a conflict between `0049` and adding `modified_date`.

The data migration in `0049` triggers the `auto_now` and attempts to write to `Report.modified_date` before it exists.